sga/templatetags/math.py

```diff
@@ -1,14 +1,21 @@
+"""
+Custom math templatetags
+"""
+
 from django import template
 
 register = template.Library()
 
+
 @register.filter
 def subtract(value, arg):
-    """ Subtracts the arg from the value """
+    """
+    Subtracts the arg from the value
+    """
     try:
         return int(value) - int(arg)
     except (ValueError, TypeError):
         try:
             return value - arg
-        except Exception:
-            return ""
\ No newline at end of file
+        except Exception:  # pylint: disable=broad-except
+            return ""
```

Comment:
Sorry, I forgot to comment on this before:
in which case `int(value) - int(arg)` (line 16) will raise an exception but `value - arg` (line 19) will work?

In general what is this supposed to do? If it has to do a numerical subtraction, then you should first check if the input is numerical type (or can be coerced to it) and then subtract or return an empty string.
BTW this implementation does not work really well if you have floats or decimal types